client/anp/talpa-video/index.ts

```diff
@@ -0,0 +1,12 @@
+import angular from 'angular';
+import { reactToAngular1 } from 'superdesk-ui-framework';
+import TalpaVideoSearchPanel from './talpaVideoSearchPanel';
+
+export default angular.module('anp.talpaVideo', [])
+  .component('talpaVideoSearchPanel', reactToAngular1(TalpaVideoSearchPanel, ['params', 'scope']))
```

Comment:
btw there is new api to register the component and avoid using that template cache,
you can see it [here](https://github.com/superdesk/superdesk-cp/pull/2)